public/app/features/explore/LiveLogs.tsx

```diff
@@ -20,15 +22,25 @@ const getStyles = (theme: GrafanaTheme) => ({
   logsRowFresh: css`
     label: logs-row-fresh;
     color: ${theme.colors.text};
-    background-color: ${selectThemeVariant({ light: theme.colors.gray6, dark: theme.colors.gray1 }, theme.type)};
+    animation: fade 2s linear 0s 1 normal forwards;
```

Comment:
Hmm shame we do not have a unique key for the rows we could do something like this:
<img width="922" alt="Screenshot 2019-09-06 at 17 19 27" src="https://user-images.githubusercontent.com/1014802/64439576-88e21580-d0ca-11e9-9584-d11e74afb57e.png">
But right now each row is unmounted on every change so not possible. @kaydelaney did you found out whether we could use labels + nanosecond timestamp as unique key for the rows?